### PR TITLE
[#5306]Improvement(jdbc-catalog): make properties required

### PR DESCRIPTION
**Pull request by lsyulong**
### What changes were proposed in this pull request?

[#5306][Improvement] Some JDBC catalog properties should be defined as required

### Why are the changes needed?

Fix: #5306 

### Does this PR introduce _any_ user-facing change?

change some JDBC catalog properties

### How was this patch tested?
No

### CODE REVIEW

File: catalogs/catalog-jdbc-common/src/main/java/org/apache/gravitino/catalog/jdbc/JdbcCatalogPropertiesMetadata.java [hunk 1]
```diff
@@ -61,24 +61,30 @@ public class JdbcCatalogPropertiesMetadata extends BaseCatalogPropertiesMetadata
                 false /* immutable */,
                 null /* defaultValue */,
                 false /* hidden */),
-            stringOptionalPropertyEntry(
+                stringPropertyEntry(
                 JdbcConfig.JDBC_DRIVER.getKey(),
                 JdbcConfig.JDBC_DRIVER.getDoc(),
+                true /* required */,
```
Comment (mchades) [inline catalogs/catalog-jdbc-common/src/main/java/org/apache/gravitino/catalog/jdbc/JdbcCatalogPropertiesMetadata.java:67]: could you plz add tests to cover the changes?

**Comment by mchades** [comment]
Run './gradlew :catalogs:catalog-jdbc-common:spotlessApply' to fix code style

**Comment by lsyulong** [comment]
> Run './gradlew :catalogs:catalog-jdbc-common:spotlessApply' to fix code style运行 './gradlew :catalogs:catalog-jdbc-common:spotlessApply' 修复代码样式

Can I use idea one click formatting?

**Comment by mchades** [comment]
> Can I use idea one click formatting?

no, need to run the command in the terminal.

**Comment by lsyulong** [comment]
> > Can I use idea one click formatting?
> 
> no, need to run the command in the terminal.

image
How should I operate

**Comment by mchades** [comment]
Sorry, it looks like you are using a Windows OS, I cannot provide advice. This command needs to be executed on Linux.

**Comment by lsyulong** [comment]
> Sorry, it looks like you are using a Windows OS, I cannot provide advice. This command needs to be executed on Linux.

image
Can I manually indent it? I haven't made any changes elsewhere

**Comment by mchades** [comment]
FYI:
https://github.com/apache/gravitino/actions/runs/11570667372/job/32207125956?pr=5337

**Comment by lsyulong** [comment]
Is there any problem with the latest submitted code? If there is, I will make the necessary modifications @mchades

**Review by mchades — APPROVED** [review]
LGTM. Thanks for your contributions!

**Pull request merged**